pkg/cli/project_update.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"fmt"
+
+	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/project"
+	"github.com/spf13/cobra"
+)
+
+func NewProjectUpdate(c CommandContext) *cobra.Command {
+	cmd := cli.Command(&ProjectUpdate{client: c.ClientFactory}, cobra.Command{
+		Use: "update [flags] PROJECT_NAME [PROJECT_NAME...]",
+		Example: `
+acorn project update my-project
+`,
+		SilenceUsage:      true,
+		Short:             "Update projects",
+		Args:              cobra.MinimumNArgs(1),
```

Comment:
If we are only allowing to update one project at a time, then this should be updated too.